package.json

```diff
@@ -2136,7 +2135,7 @@
         "named-js-regexp": "^1.3.3",
         "node-fetch": "^2.6.7",
         "node-stream-zip": "^1.6.0",
-        "pdfkit": "^0.12.1",
+        "pdfkit": "^0.13.0",
```

Comment:
Had to udpate as it was pulling in some stuff that cased ` npm list --production --parseable --depth=99999 --loglevel=error` to fail.